dot/network/mock_transaction_handler.go

```diff
@@ -12,20 +12,20 @@ type MockTransactionHandler struct {
 	mock.Mock
 }
 
-// HandleTransactionMessage provides a mock function with given fields: _a0
-func (_m *MockTransactionHandler) HandleTransactionMessage(_ peer.ID, _a0 *TransactionMessage) (bool, error) {
-	ret := _m.Called(_a0)
+// HandleTransactionMessage provides a mock function with given fields: _a0, _a1
+func (_m *MockTransactionHandler) HandleTransactionMessage(_a0 peer.ID, _a1 *TransactionMessage) (bool, error) {
+	ret := _m.Called(_a0, _a1)
 
 	var r0 bool
-	if rf, ok := ret.Get(0).(func(*TransactionMessage) bool); ok {
-		r0 = rf(_a0)
+	if rf, ok := ret.Get(0).(func(peer.ID, *TransactionMessage) bool); ok {
+		r0 = rf(_a0, _a1)
 	} else {
 		r0 = ret.Get(0).(bool)
 	}
 
 	var r1 error
-	if rf, ok := ret.Get(1).(func(*TransactionMessage) error); ok {
-		r1 = rf(_a0)
+	if rf, ok := ret.Get(1).(func(peer.ID, *TransactionMessage) error); ok {
+		r1 = rf(_a0, _a1)
```

Comment:
That breaks some tests, I've updated the mocks and fixed the tests in #2062 
Either apply the same fix as in this other PR, or wait for it to be merged and then rebase on development / merge development in your branch.